### DISCUSSION TITLE
Change font colour in dp.Table

### DISCUSSION THREAD

**Discussion by ashprice** [datapane/datapane#160] [Help]
Hello,
I would like a dark-themed report. 

Currently, I have my graphs set to use the plotly_dark theme. I went and set the dp.ReportFormatting to use black. There is a slight mismatch. I suppose I could hard-code those colors, though I wonder if anyone knows how to set this all up to the plotly_dark palette?

Anyway - my main question is this: when changing the background to black, the data in a table stays black as well. How can I change the dp.Table formatting so that the font is white? I tried including a HTML block without much luck.

**Comment by johnmicahreid** [comment]
Hi @ashprice, 

You can try setting the styling to use light prose: https://docs.datapane.com/reports/configuring-reports/styling

Here's an example: 

```python
report = dp.Report(...)

report.upload(name="test_report", formatting=dp.ReportFormatting(light_prose=True))
```

Let us know if that works!
